tensorflow/lite/micro/tools/ci_build/test_project_generation.sh

```diff
@@ -22,15 +22,14 @@ cd "${ROOT_DIR}"
 
 source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
 
+EXAMPLES="-e hello_world -e magic_wand -e micro_speech -e person_detection"
+
 TEST_OUTPUT_DIR=$(mktemp -d)
 
-readable_run \
+#readable_run \
```

Comment:
Just confirming this comment is intended? It's a bit surprising to have a comment with a \ continuation.

Reply:
Fixed. This was left behind from my testing.